### PR TITLE
files-to-epub@elblake: Initial release

### PR DESCRIPTION
**Pull request by elblake**
Hello,

I've made an action for selecting plain text and images in a folder to turn them into a EPUB file.

A change was made to the makepot script for perl scripts, but I made it a separate commit
to make it easier to find in the history. Let me know if there needs to be any changes or if
there are any issues.

Thanks,
Edward Blake

### CODE REVIEW

File: cinnamon-spices-makepot [hunk 1]
```diff
@@ -253,6 +253,15 @@ def make_pot(uuid: str) -> CapturedOutput:
                                      cwd=output_dir)
         os.chmod(output_dir.joinpath(shell_file), 0o0755)
 
+    # Extract translatable strings from perl scripts
+    perl_list = glob('**/*.pl', recursive=True, root_dir=output_dir)
+    for perl_file in perl_list:
+        output += get_command_output(['xgettext', '-k_', '-jL', 'Perl',
```
Comment (rcalixte) [inline cinnamon-spices-makepot:259]:
```suggestion
        output += get_command_output(['xgettext', '-jL', 'Perl',
```


Do you need to specify the keyword parameter here?

**Comment by elblake** [comment]
I got a notification of a comment earlier but I can't seem to find it. I've added the keyword switch as it seems xgettext defaults to 'gettext "string"' for perl source.

**Comment by rcalixte** [comment]
That was me. Just something that I saw initially. I'll get around to reviewing this if no one else has a chance. I need to be in a certain mindset before I can read a few thousand lines of Perl. 😅

**Comment by elblake** [comment]
Haha that's fine thanks, I write perl code in a style that mostly looks like most other curly languages if that helps.

**Comment by elblake** [comment]
What I could do is refactor it a bit and split it up into a few files.

**Comment by elblake** [comment]
I feel I have to rethink how this was made so I'll be closing this pull request for the time being. And the recent change about the licensing adds a layer of ambiguity that gives me a bit of anxiety which is highlighted by the current wordpress debacle that is happening at the moment. Which I feel validates my reluctance to put significant contributions under other people's licenses.

**Comment by elblake** [comment]
I just wanted to mention to please disregard everything I wrote in the previous comment, I'll try to find time to refactor the action. My apologies and thanks for doing the reviews.

**Pull request closed without merging**